src/components/github-htmlview.component.js

```diff
@@ -290,6 +297,48 @@ export class GithubHtmlView extends Component {
 
           return undefined;
         },
+        details: (node, index, siblings, parent, defaultRenderer) => {
+          const detailsChildren = node.children || [];
+          const childrendComponent = [];
+          let summaryText = 'details';
+
+          detailsChildren.forEach((child, childIdx) => {
+            if (child.type === 'tag' && child.name === 'summary') {
+              const summaryChild = child.children[0] || {};
+
+              summaryText = summaryChild.data || 'details';
```

Comment:
The default is redundant with `summaryText` declaration.

Also, summaryChild may need additional treatment (if it's an image for example). It needs to be rendered()

Reply:
I guess I was too hasty, didn't consider other cases of summaryChild. Thank you so much for your patient 😄

Reply:
No problems, I did make the same mistake quiet a few times before getting on pace :)
I'm available in [gitter](https://gitter.im/git-point/Lobby) today if you need some help!